demo-app/src/pages/ot/config-builder/index.tsx

```diff
@@ -1,35 +1,6 @@
-import RenderConfig from '@/components/RenderConfig'
-import { useState } from 'react'
-import ComponentList from './ComponentList'
-import ConfigYaml from './ConfigYaml'
-import {
-  CONFIG_BASE,
-  ConfigBuilderContext,
-  INITIAL_FRAMES_PATH,
-} from './config-builder-context'
+import { ConfigBuilderPage } from '@open-truss/open-truss/client'
+import OT_COMPONENTS from '@/lib/ot-components'
 
-export default function ConfigBuilderPage(): React.JSX.Element {
-  const [config, setConfig] = useState<string>(CONFIG_BASE)
-  const [framesPath, setFramesPath] = useState<string>(INITIAL_FRAMES_PATH)
-
-  return (
-    <ConfigBuilderContext.Provider
-      value={{ config, setConfig, framesPath, setFramesPath }}
-    >
-      <div
-        style={{
-          display: 'flex',
-          justifyContent: 'space-between',
-          padding: 10,
-        }}
-      >
-        <div>
-          <ComponentList />
-          <h2>config</h2>
-          <ConfigYaml config={config} />
-        </div>
-        <div>{config !== CONFIG_BASE && <RenderConfig config={config} />}</div>
-      </div>
-    </ConfigBuilderContext.Provider>
-  )
+export default function ConfigBuilder(): React.JSX.Element {
+  return <ConfigBuilderPage components={OT_COMPONENTS} />
```

Comment:
Yes! I love how this is a NextJS page but OT itself is agnostic and could be rendered using any framework ✨ 

I'm not sure why but the `*Page` suffix isn't my favorite. I'll noodle on that